GameOfLife/GameOfLife/FileIO.cs

```diff
@@ -0,0 +1,137 @@
+﻿using GameOfLife.Interfaces;
+using GameOfLife.Models;
+
+namespace GameOfLife
+{
+    public class FileIO : IFileIO
+    {
+        private string _filePath = @"C:\Users\sergejs.hramkovs\OneDrive - Accenture\Documents\field.txt";
```

Comment:
choose folder so others can run it too

Reply:
Changed the filepath and added directory creation if it doesn't exist.